www/search

```diff
@@ -1226,6 +1227,10 @@ else if ($term || $browse)
 
         echo "<i>No results were found.</i>";
 
+        if ($games_filtered == true) {
+            echo '<p><i>A filter was applied to your search. <a href="search?searchfor=' . $term . '&nofilter=1">Click</a> to search again without the filter.</i></p>';
```

Comment:
This breaks when using `/search?browse`. In that case, there are no terms, and clicking the link takes you to `/search?searchfor=&nofilter=1` without browse mode, showing you the Advanced Search Tips instead of unfiltered browse results.

Reply:
Ok, I tried to fix this.